src/Simplex/Chat/Controller.hs

```diff
@@ -293,7 +293,7 @@ data ChatCommand
   | APICreateChatItem {noteFolderId :: NoteFolderId, composedMessage :: ComposedMessage}
   | APIUpdateChatItem {chatRef :: ChatRef, chatItemId :: ChatItemId, liveMessage :: Bool, msgContent :: MsgContent}
   | APIDeleteChatItem ChatRef (NonEmpty ChatItemId) CIDeleteMode
-  | APIDeleteMemberChatItem GroupId (NonEmpty (GroupMemberId, ChatItemId))
+  | APIDeleteMemberChatItem GroupId (NonEmpty ChatItemId)
```

Comment:
GroupMemberId wasn't required for this api - it was only used to make a redundant check that retrieved chat item's chatDir is for expected group member (with UI passing member id taken from chat item's chatDir). It was cumbersome to replicate in batch implementation and unnecessary